python/arcticdb/adapters/lmdb_library_adapter.py

```diff
@@ -162,14 +161,12 @@ def create_library(self, name, library_options: LibraryOptions):
 
         return lib
 
-    def cleanup_library(self, library_name: str, library_config: LibraryConfig):
-        for k, v in library_config.storage_by_id.items():
-            lmdb_config = LmdbConfig()
-            v.config.Unpack(lmdb_config)
-            shutil.rmtree(os.path.join(lmdb_config.path, library_name), onerror=_rmtree_errorhandler)
+    def cleanup_library(self, library_name: str):
+        shutil.rmtree(os.path.join(self._path, library_name), onerror=_rmtree_errorhandler)
```

Comment:
Naive question: should we add validation on the value of `self._path` to make sure this can't be used to delete some folders on the filesystem?

Reply:
That is a good point isn't it, we could have logic like:

- Look at `self._path / library_name`
- Does it contain .mdb files?
** No? Stop.
- Delete the .mdb files
- Is the dir now empty?
** Yes? Delete the dir.
** No? Stop.

Reply:
Raised #906 to do that, to avoid scope creep in this PR.